src/main/java/com/rental/rentalApp/RentalAppApplication.java

```diff
@@ -1,7 +1,11 @@
 package com.rental.rentalApp;
 
 import com.rental.rentalApp.entities.Client;
+import com.rental.rentalApp.entities.PaymentMethod;
+import com.rental.rentalApp.entities.VehicleType;
 import com.rental.rentalApp.repositories.ClientRepository;
+import com.rental.rentalApp.repositories.PaymentMethodRepository;
+import com.rental.rentalApp.repositories.VehicleTypeRepository;
```

Comment:
Please check if this works

```suggestion
import com.rental.rentalApp.entities.*;
import com.rental.rentalApp.repositories.*;
```